dcicutils/env_utils.py

```diff
@@ -425,11 +426,13 @@ def fresh_cgap_deployed_state_for_testing(cls):
             yield
 
     @classmethod
-    def app_case(self, *, if_cgap, if_fourfront):
+    def app_case(self, *, if_cgap, if_fourfront, if_smaht=None):
         if EnvUtils.ORCHESTRATED_APP == APP_FOURFRONT:
             return if_fourfront
         elif EnvUtils.ORCHESTRATED_APP == APP_CGAP:
             return if_cgap
+        elif EnvUtils.ORCHESTRATED_APP == APP_SMAHT:  # = None to avoid incompatibility
```

Comment:
You might alternatively want to initialize it before the `if` by doing:
```
if if_smaht is None:
    if_smaht = if_cgap
```
on the assumption that the cgap-like code is the best thing to use?

It might be prudent to make a similar change to `_orchestrated_app_case` in `env_utils_legacy.py`. It might not be reachable code, but it probably can't hurt.